docs/framework/Release-Notes/01-January-Preview-of-Quality-Rollup

```diff
@@ -0,0 +1,59 @@
+<h2>January Preview of Quality Rollup</h2>
```

Comment:
Can we convert this to markdown? We really shouldn't allow HTML anywhere on this repo, certainly not as the primary document format.

Reply:
Did some quick parsing - https://github.com/dotnet/docs/pull/40186